### PR TITLE
dist: Update NEWS with updates since rc3

### PR DESCRIPTION
**Pull request by bwbarrett**
Signed-off-by: Brian Barrett <bbarrett@amazon.com>

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -55,20 +55,22 @@ included in the vX.Y.Z section and be denoted as:
    (** also appeared: A.B.C)  -- indicating that this item was previously
                                  included in release version vA.B.C.
 
-3.1.1 -- TBD
-------------
+3.1.0 -- March, 2018
```
Comment (jsquyres) [inline NEWS]: Should this now be April 2018?

Comment (bwbarrett) [reply NEWS]: Blah. Will fix.

**Comment by bwbarrett** [comment]
@jsquyres can you take another look?

**Comment by ibm-ompi** [comment]
The IBM CI (PGI Compiler) build failed! Please review the log, linked below.

 Gist: https://gist.github.com/ibm-ompi/88b480fb98dfd2bdac939f023e1f8fe6

**Comment by jjhursey** [comment]
bot:ibm:retest

**Pull request merged**